lib/et/src/et.app.src

```diff
@@ -31,7 +31,8 @@
     et_wx_viewer
    ]},
   {registered, [et_collector]},
-  {applications, [stdlib, kernel]},
+  {applications, [stdlib, kernel, wx, runtime_tools]},
+  {optional_applications, [wx, runtime_tools]},
```

Comment:
wx should not  be in applications.

Reply:
@IngelaAndin `optional_applications` should be a subset of `applications`. This was a discussion back to when the feature was implemented, because we need `applications` to still provide a consistent ordering (as in #2675).

Reply:
@josevalim Humm ... I guess I misread the documentation.  @dgud promised he would have a look at this as he has worked more with the applications at hand.